### ISSUE TITLE
License issues

### ISSUE THREAD

**Issue by Jagger0083** [Masuzu/SarasaBot#157]
As the title says. I made sure the key file is in the folder and the serial number matches the one on my account page but it's still saying that I need to get a new license

**Comment by Masuzu** [comment]
Hello,
The reference serial number which should match should be the one which is created when you start SarasaBot.exe, written in a file named "serial_number.txt" in the same folder.
There were past reports of discrepancies between the serial number generated by the generator and the one actually used.

**Comment by Jagger0083** [comment]
Yeah, I'm still seeing the same issue. All 3 match (Web account, serial_number.txt, and in  the serial generator in folder)

**Comment by Masuzu** [comment]
In that case, did you redownload your key.txt file?
